ajdoc/pom.xml

```diff
@@ -47,12 +48,6 @@
       <artifactId>org.aspectj.ajdt.core</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <!-- enables easy dependency on tools.jar -->
-      <groupId>com.github.olivergondza</groupId>
```

Comment:
I get nervous when a stable product depends upon a 0.1 version (not 1.0) of a project on GitHub I don't know anything about.

Reply:
It didn't make you nervous more than two years ago when you agreed it in https://github.com/eclipse/org.aspectj/commit/bd6a4b2b. I just moved the block around to another place in the POM. 😉

Reply:
I never even looked at the dependency. Actually, it is just a POM [with a profiles](https://github.com/olivergondza/maven-jdk-tools-wrapper/blob/a1c5c71590a8de54376e2155ae0141e220116dec/pom.xml#L67-L109). If it makes you feel better, we can inline them into our own JAR.

Reply:
I obviously didn't review it properly then. If its just a refactoring/rearrange, I'm more comfortable yes.